### PR TITLE
zh_TW Translation added

### PR DESCRIPTION
**Pull request by niotw**
zh_TW Translation added

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -56,3 +56,7 @@
 
 - Ömür Turan ([@omurturan](https://github.com/omurturan))
 - Serkan Algur ([@serkanalgur](https://github.com/serkanalgur))
+
+### Traditional Chinese
+
+- [@Nio] (https://github.com/niotw)
```
Comment (rugk) [inline CONTRIBUTORS.md:62]: If you look at the Markdown preview, the space here prevents that from rendering correctly…

```suggestion
- [@Nio](https://github.com/niotw)
```

**Comment by rugk** [comment]
Hi @niotw,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Also notice there are more things to translate like the wiki e.g. etc. [See the contributing guide for more information.](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-add-on-listings)

Please subscribe to #138 to get notified for new updates on translations etc. :smiley:

As for the credit, just assure you've used no source from other authors. As @Cye3s already created a zh-CN translation in https://github.com/rugk/offline-qr-code/pull/246/. If you've used that and e.g. transcopied/converted that, it's better to mention them in the appropriate locations too. [See this section](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on) for our translation credit guidelines.

**Comment by rugk** [comment]
Please do also note one _important_ precautionary warning of me: In case you did not notice, you have committed (i.e. created commits) with potentially sensitive data like your (full/private) name or mail.
Anyone, who clones your fork/this repo can view these private details, exactly as you've committed them. Also, if this PR gets merged, this will stay in the (commit) history of this project and will thus hard to be removed later.

So if this was unintentional and you want that data to be changed/removed first, better do it now. :smiley: [Here is a good resource](https://stackoverflow.com/a/750182/5008962) ([actually from GitHub's help](https://help.github.com/en/articles/changing-author-info)) on how to change your name/mail afterwards.
Also note that when setting your git [username](https://help.github.com/en/articles/setting-your-username-in-git) and [especially your email](https://help.github.com/en/articles/setting-your-commit-email-address), you can actually write/choose anything there. As for GitHub, it is recommend to use your GitHub username and – if you want to keep your mail private – the GitHub provided `noreply` address. You can [even block pushes of commits with private mails](https://help.github.com/en/articles/blocking-command-line-pushes-that-expose-your-personal-email-address).

So far just the possibilities. I just wanted to serve this as a friendly reminder, so you don't notice this only when it's too late. What you'll do depends solely on you. :smile:

File: CONTRIBUTORS.md (hunk 1 above)
Comment (niotw) [reply CONTRIBUTORS.md:62]: Okay, modify it now.

**Comment by niotw** [comment]
Yes it's can be merged

File: src/_locales/zh_TW/messages.json [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+    "description": "Button for the tip shown to remind the user to rate the addon.",
+    "hash": "821d7f942b02e2b56cd0424d3ab65de8"
+  },
+  "tipSaveQrCode": {
+    "message": "您知道可以保存二維條碼嗎？",
+    "description": "A tip shown to get the user to find the save option.",
+    "hash": "84f0a0027a58c98d9158ab9894fe8427"
+  },
+  "tipSaveQrCodeLink": {
+    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ#how-to-save-the-qr-code-on-disk",
```
Comment (rugk) [inline src/_locales/zh_TW/messages.json:145]: Well, as you seem to have that translated in the wiki now, you can change the link to the new wiki page.

(Please do this for all links.)

Ref:
> Do not forget that files in the project's main repository itself often refer to the wiki, so remember to also replace the links in here.

https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-wiki

File: src/_locales/zh_TW/messages.json [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+    },
+    "hash": "717944c73c8d92595af6c07bb960b52d"
+  },
+  "optionQrQuietZoneDescr": {
+    "message": "二維條碼四週的空白塊數量，顯示為背景色。",
+    "description": "The description of the quiet zone option.",
+    "hash": "94a93f5a15d9303e8003eae8c0b94ed1"
+  },
+  "optionQrQuietZoneDescrLink": {
+    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ#what-is-a-quiet-zone",
```
Comment (rugk) [inline src/_locales/zh_TW/messages.json:357]: same here

File: src/_locales/zh_TW/messages.json (hunk 2 above)
Comment (niotw) [reply src/_locales/zh_TW/messages.json]: Edited.

File: src/_locales/zh_TW/messages.json (hunk 3 above)
Comment (niotw) [reply src/_locales/zh_TW/messages.json]: edited.

File: src/_locales/zh_TW/messages.json [hunk 4]
```diff
@@ -354,7 +354,7 @@
     "hash": "94a93f5a15d9303e8003eae8c0b94ed1"
   },
   "optionQrQuietZoneDescrLink": {
-    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ#what-is-a-quiet-zone",
+    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ-(chinese)#what-is-a-quiet-zone",
```
Comment (rugk) [inline src/_locales/zh_TW/messages.json:357]:
```suggestion
    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ-(chinese)#%E4%BB%80%E9%BA%BC%E6%98%AF-quiet-zone",
```

File: src/_locales/zh_TW/messages.json [hunk 5]
```diff
@@ -142,7 +142,7 @@
     "hash": "84f0a0027a58c98d9158ab9894fe8427"
   },
   "tipSaveQrCodeLink": {
-    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ#how-to-save-the-qr-code-on-disk",
+    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ-(chinese)#how-to-save-the-qr-code-on-disk",
```
Comment (rugk) [inline src/_locales/zh_TW/messages.json:145]:
```suggestion
    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ-(chinese)#%E5%A6%82%E4%BD%95%E5%B0%87%E4%BA%8C%E7%B6%AD%E6%A2%9D%E7%A2%BC%E5%84%B2%E5%AD%98%E5%88%B0%E7%A3%81%E7%A2%9F%E4%B8%AD",
```

**Pull request merged**